client/components/telecom/telephony/alias/liveCalls/telecom-telephony-alias-configuration-liveCalls.html

```diff
@@ -45,104 +45,65 @@ <h2 class="widget-presentation-title" data-translate="telephony_alias_configurat
-                        </button>
-                    </td>
-                </tr>
-            </tbody>
-            <tbody data-ng-if="$ctrl.isLoading">
-                <tr>
-                    <td colspan="5">
-                        <div class="text-center">
-                            <oui-spinner></oui-spinner>
-                        </div>
-                    </td>
-                </tr>
-            </tbody>
-            <tbody data-ng-if="!$ctrl.isLoading && !$ctrl.calls.length">
-                <tr>
-                    <td colspan="5"
-                        class="text-center"
-                        data-translate="telephony_alias_configuration_mode_calls_empty">
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-        <div class="mt-2 mb-3"
-             data-ng-show="!$ctrl.isLoading && $ctrl.calls.length"
-             data-pagination-front
-             data-items="$ctrl.calls"
-             data-paginated-items="$ctrl.callsPaginated"
-             data-current-page="currentPage"
-             data-nb-pages="nbPages"
-             data-items-per-page="itemsPerPage"
-             data-page-placeholder="{{ 'common_pagination_page' | translate: { currentPage:currentPage, nbPages:nbPages } }}"
-             data-item-per-page-placeholder="{{ 'common_pagination_display' | translate }}">
-        </div>
-    </div>
+
+
+    <oui-datagrid data-rows="$ctrl.callsPaginated">
+
+        <oui-column data-title="'telephony_alias_configuration_mode_calls_number' | translate"
+                    data-property="callerIdNumber" data-sortable="asc">
+            <span class="text-nowrap mr-3 text-capitalize"
+                  data-ng-bind="$row.callerIdNumber | phoneNumber"></span>
+            <span class="text-nowrap mr-3"
+                  data-ng-if="$row.callerIdName"
+                  data-ng-bind="'(' + $row.callerIdName + ')'"></span>
+        </oui-column>
+
+        <oui-column data-title="'telephony_alias_configuration_mode_calls_queue_time' | translate"
+                    data-property="begin" data-sortable>
+            <elapsed-time data-from="$row.begin"></elapsed-time>
+        </oui-column>
+
+        <oui-column data-title="'telephony_alias_configuration_mode_calls_time' | translate"
+                    data-property="answered" data-sortable>
+            <span class="text-nowrap mr-3 text-capitalize"
+                  data-ng-if="!$row.answered"
+                  data-ng-bind="$row.answered ? 'yes' : 'no' | translate"></span>
+            <elapsed-time data-ng-if="$row.answered"
+                          data-from="$row.answered"></elapsed-time>
+        </oui-column>
+
+        <oui-column data-title="'telephony_alias_configuration_mode_calls_agent' | translate"
+                    data-property="agent" data-sortable>
+            <span class="text-nowrap mr-3" data-ng-bind="$row.agent || '-'"></span>
```

Comment:
il faudrait être en mesure de ne plus utiliser ces classes ovh-common-style (text-nowrap, mr-3, word-break)

Reply:
`text-nowrap` and `mr-3` are no part of _ ovh-common-style_ but _bootstrap 4_.